equinox/serialisation.py

```diff
@@ -2,11 +2,11 @@
 from typing import Any, BinaryIO, Callable, Union
 
 import jax.numpy as jnp
-import jax.tree_util as jtu
 import numpy as np
 
 from . import experimental
 from .custom_types import PyTree
+from .tree import _ordered_tree_map
```

Comment:
Likewise we shouldn't be importing anything private (with the underscore). This indicates that it's private to the module that defined it.

Reply:
Defining it now in `serialisation.py`.